src/cli/index.spec.ts

```diff
@@ -53,4 +67,18 @@ describe("cli", () => {
       "
     `);
   });
+
+  it("should ignore empty spaces when using positional argument", async () => {
+    const execSyncMock = jest.requireMock("child_process").execSync;
+    mockFs();
+    await run(["node", "swa", "build", "   app  ", "--app-build-command", "npm run something"]);
+    expect(execSyncMock.mock.calls[0][1].cwd).toBe("app");
+  });
+
+  it("should not interpret empty spaces as a positional argument", async () => {
+    const execSyncMock = jest.requireMock("child_process").execSync;
+    mockFs();
+    await run(["node", "swa", "build", "    ", "--app-build-command", "npm run something", "   "]);
```

Comment:
Great thanks! @sinedied  for adding the test for `swa build`.. 

Just to confirm if these tests cover the changes we did in deploy.ts/start.ts if not will it make sense for us to add separate tests for those scenarios ?

Reply:
The added logic is the same for build/start/deploy, so it should validate these changes, but not cover it in case of refactor of start/deploy.

Currently there's no tests (at all) for the cli commands deploy/start/login, hence why I reused the mocks made for the build command tests. Adding minimal tests for these commands would be useful, though not the scope of this PR.